app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -811,8 +811,9 @@ def self.vm_inv_to_hardware_hash(inv)
     result[:bios] = bios unless bios.blank?
 
     if inv["numCpu"].present?
-      result[:numvcpus] = inv["numCpu"].to_i
-      result[:cores_per_socket], result[:logical_cpus] = calculate_cores_and_sockets(inv["numCpu"], config.try(:fetch_path, "hardware", "numCoresPerSocket"))
+      result[:logical_cpus] = inv["numCpu"].to_i
```

Comment:
Yes numCPU is the total number of logical vCPUs.

If I create a VM with 2 sockets and 2 cores per socket, the result is this from the API:
numCPU	int	4
numCoresPerSocket	int	2